PurchasesTests/Purchasing/ProductInfoExtractorTests.swift

```diff
@@ -0,0 +1,194 @@
+import XCTest
+import Nimble
+
+import Purchases
+
+class ProductInfoExtractorTests: XCTestCase {
+
+    func testExtractInfoFromProductExtractsProductIdentifier() {
+        let product = MockSKProduct(mockProductIdentifier: "cool_product")
+        let productInfoExtractor = RCProductInfoExtractor()
+
+        let receivedProductInfo = productInfoExtractor.extractInfo(from: product)
+
+        expect(receivedProductInfo.productIdentifier) == "cool_product"
+    }
+
+    func testExtractInfoFromProductExtractsPrice() {
+        let product = MockSKProduct(mockProductIdentifier: "cool_product")
+        product.mockPrice = NSDecimalNumber(decimal: 10.99)
```

Comment:
same here, I would have `let expected = 10.99`. This could apply to all the tests in this file